x-pack/plugins/ml/public/application/components/import_export_jobs/export_jobs_flyout/export_job_warning_callout.tsx

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+import { EuiCallOut, EuiText, EuiAccordion, EuiSpacer } from '@elastic/eui';
+import type { JobDependencies } from './jobs_export_service';
+
+interface Props {
+  jobs: JobDependencies;
+}
+
+export const ExportJobDependenciesWarningCallout: FC<Props> = ({ jobs: allJobs }) => {
+  const [jobs, jobsWithCalendars, jobsWithFilters] = filterJobs(allJobs);
+  const usingCalendars = jobsWithCalendars.length > 0;
+  const usingFilters = jobsWithFilters.length > 0;
+
+  if (usingCalendars === false && usingFilters === false) {
+    return null;
+  }
+
+  return (
+    <>
+      <EuiCallOut
+        title={getTitle(jobs, jobsWithCalendars.length, jobsWithFilters.length)}
+        color="warning"
+      >
+        <FormattedMessage
+          id="xpack.ml.importExport.exportFlyout.exportJobDependenciesWarningCallout.calendarDependencies"
+          defaultMessage="When importing jobs into a new environment, calendars and filters which are relied upon by jobs will need to be recreated."
```

Comment:
Might be better to use `filter lists` here as that is the term we use in the UI. Also I'd go with something like `the calendars and filters which are used by these jobs will need to be recreated first.`

Reply:
This seems to be the one place where we're describing why there's a problem and what to do about it. My advice is to give more context and details, for example:

> When you export jobs, calendars and filter lists are not included. You must create the filter lists before you import a job; otherwise, it fails. If you want the new job to continue to ignore scheduled events, you must create the calendars.

Reply:
Updated in https://github.com/elastic/kibana/pull/107416/commits/e64c591042b919577dc0e9c81c1ed2e92b043579